src/enums/nameChangeOptions.ts

```diff
@@ -0,0 +1,5 @@
+export enum NameChangeOptions {
+  CORRECT_NAME = 'correct-name',
+  CORRECT_NAME_TO_NUMBER = 'correct-name-to-number',
+  CORRECT_NEW_NR = 'correct-new-nr'
+}
```

Comment:
Don't add this file -- it's already imported from the shared components repo. See the index file in this folder.